external/libtww/d/save/d_save.h

```diff
@@ -426,11 +427,15 @@ static_assert(sizeof(dSv_player_c) == 0x380);
 
 class dSv_memBit_c {
 public:
+    void setKeyNum(int num) { mKeyNum = num; }
+    u8& getKeyNum() { return mKeyNum; }
+    void offDungeonItem(int flag) { mDungeonItem &= ~(1 << flag); }
+ //   void onDungeonSwitch(int flag) { mSwitch[flag >> 5] = mSwitch[flag >> 5] | 1 << (flag & 0x1F); }

```

Comment:
Is this code you don't want anymore?

Reply:
This is code that does not currently do what it should, possibly due to memory being misaligned or some other problem I am unaware of. It is commented out for now to reflect the fact that it does not work.